react/ShelfContent.js

```diff
@@ -76,27 +78,35 @@ class ShelfContent extends Component {
   }
 
   get itemsToShow() {
-    const { itemsPerPage, width, gap } = this.props
-    const maxItems = Math.floor(width / (DEFAULT_SHELF_ITEM_WIDTH + gap))
+    const { itemsPerPage, width } = this.props
+    const maxItems = Math.floor(width / (DEFAULT_SHELF_ITEM_WIDTH))
```

Comment:
Removing the gap from the math won't make the possibility to have more itens that can fit in the window?
Like, could only fit 3 but somehow got 4 from this math?

Reply:
I know you changed the gap to be a tachion-like prop, but you can map it to the width and use in the math.

Reply:
Gap now are paddings(before are margins),  so this we do not have this problem.